strategy/rfp-to-videographers.md

```diff
@@ -1,154 +1,114 @@
 [Name]
 [Studio Name]
+[Address]
 
-Dear [Name],
+Dear [Name]:
 
-**Re. Request for proposal to produce a short video commissioned by the Humane Tech Community**
+**Re: Request for proposal to produce a short video commissioned by the Humane Tech Community**
 
-The Humane Tech Community ("HTC"), founded by The Center for Humane Technology (www.humanetech.com), is a grassroots movement whose purpose is to "Re-align Technology with Humanity's Best Interests". Based upon four pillars, namely Awareness, Regulation, Participate and Realization, members of the community are actively involved in numerous initiatives aimed at challenging the status quo.
+The Humane Tech Community ("HTC"), founded by The Center for Humane Technology (www.humanetech.com), is a grassroots movement whose purpose is to "Re-align Technology with Humanity's Best Interests". Based upon four pillars---namely, Awareness, Regulation, Participation, and Realization---the community is actively involved in numerous initiatives aimed at changing the status quo and strengthening society.
```

Comment:
```suggestion
The Humane Tech Community ("HTC"), founded by The Center for Humane Technology (www.humanetech.com), is a grassroots movement whose purpose is to "Re-align Technology with Humanity's Best Interests". Based upon four pillars - namely, Awareness, Regulation, Participation, and Realization - the community is actively involved in numerous initiatives aimed at changing the status quo and strengthening society.
```

Probably your text editor long dashes are translated into 3 individual dashes in Markdown. Let's have them single dashes and spaces in the markdown template.